qlib/kernel/mod.rs

```diff
@@ -62,12 +62,16 @@ pub static TSC: Tsc = Tsc::New();
 pub static SHARESPACE: ShareSpaceRef = ShareSpaceRef::New();
 pub static IOURING: IOUringRef = IOUringRef::New();
 pub static KERNEL_PAGETABLE: Singleton<PageTables> = Singleton::<PageTables>::New();
-// used for allocate new page table see pub fn Fork(&self, pagePool: &PageMgr) -> Result<Self> {
-pub static PAGE_MGR: PageMgrRef = PageMgrRef::New();    
+
+ 
 pub static LOADER: Singleton<Loader> = Singleton::<Loader>::New();
 pub static WAIT_CONTAINER_FD: AtomicI32 = AtomicI32::new(-1);
+
+// place holder, will never be used by qvisor
 pub static KERNEL_STACK_ALLOCATOR: Singleton<AlignedAllocator> =
     Singleton::<AlignedAllocator>::New();
+// place holder, will never be used by qvisor
+pub static PAGE_MGR: PageMgrRef = PageMgrRef::New();   
```

Comment:
if never used by qvisor, why should these be defined by under qlib/ instead of qkernel?

Are there unresolved reference?

Reply:
After refactoring, Qvisor does not need to use `PAGE_MGR` and  `KERNEL_STACK_ALLOCATOR` anymore.    But the code should still be in qlib because we want to merge cc branch into main.

Reply:
Switches will be added later to make cc compatible with upstream